mem/log.go

```diff
@@ -117,15 +119,26 @@ func (log *memlog) Query(specs ...margaret.QuerySpec) (luigi.Source, error) {
 		}
 	}
 
+	if qry.reverse && qry.live {
+		return nil, errors.Errorf("memlog: can't do reverse and live")
```

Comment:
This error message feels vague for someone unfamiliar with the code.

"can't do reverse and live (what?)"

How about "memlog: reverse and live are incompatible query parameters"?